docs/docs/concepts/architecture.md

```diff
@@ -76,6 +80,18 @@ Cons:
 
 - Potential data-loss and the clients do not get a chance to retry/resend the events. The possiblity of data-loss occurs when the kafka borker cluster is facing a downtime.
 
+### EVENT_ACK = 1
+
+Raccoon sends the acknowledgments after events are produced to Kafka.
```

Comment:
`after the events are acknowledged successfully from the Kafka brokers`.

I think there should be guidance & explanation here in terms of in what cases should these options be chosen. Sycn: enables reliable end-to-end acknowledgements where the underlying publisher acknowledgement is leveraged.

Reply:
Updated